meson.build

```diff
@@ -1,12 +1,15 @@
 project('vkcube', ['c'], version : '0.1', license : 'MIT',
         default_options : ['c_std=c11'])
 
+cc = meson.get_compiler('c')
+
 dep_libdrm = dependency('libdrm')
 dep_gbm = dependency('gbm')
 dep_libpng = dependency('libpng')
 dep_wayland = dependency('wayland-client')
 dep_xcb = dependency('xcb')
 dep_vulkan = dependency('vulkan')
+dep_m = cc.find_library('m', required : false)
```

Comment:
Why are you saying required: false?  I think "m" should always be there but still, maybe true would be better?